Evaluator/TA/trend_evaluator/resources/DeathAndGoldenCrossEvaluator.md

```diff
@@ -0,0 +1,3 @@
+DeathAndGoldenCrossEvaluator is based on two [moving averages](https://www.investopedia.com/terms/m/movingaverage.asp), basically one of **50** periods and other one of **200**.
+
+When the fast moving average cross above the slow moving average it's called a [Golden Cross](https://www.investopedia.com/terms/g/goldencross.asp) and when it's the slow which cross above the fast it's a [Death Cross](https://www.investopedia.com/terms/d/deathcross.asp)
```

Comment:
I think we should add something like:
> Produces a `-1` signal when the fast moving average is above the slow (bull market) one and `1` when the slow moving average is above the fast one (bear market)

Reply:
```suggestion
Suggested change 
When the fast moving average cross above the slow moving average(**-1**) this indicate a bull market and it's called a [Golden Cross](https://www.investopedia.com/terms/g/goldencross.asp). When it's the slow which cross above the fast(**1**) it's a [Death Cross](https://www.investopedia.com/terms/d/deathcross.asp) which mean it's a bear market.

```

Reply:
👍 I think it's also good to say that -1 and 1 are permanent signals, not just when a cross is hapening

Reply:
like this ?

```suggestion
If the fast moving average is above the slow moving average this indicate a bull market(**-1**) and when this happened it's called a [Golden Cross](https://www.investopedia.com/terms/g/goldencross.asp).
Inversely, if it's the fast moving average is above the slow moving average this indicate a bear market(**1**) and when this happened it's called a [Death Cross](https://www.investopedia.com/terms/d/deathcross.asp).
```

Reply:
With minor improvements:

If the fast moving average is above the slow moving average, this indicates a bull market (signal: **-1**) When this happens it's called a [Golden Cross](https://www.investopedia.com/terms/g/goldencross.asp).
Inversely, if it's the fast moving average is above the slow moving average this indicates a bear market (signal: **1**). When this happens it's called a [Death Cross](https://www.investopedia.com/terms/d/deathcross.asp).